### ISSUE TITLE
AUR Package

### ISSUE THREAD

**Issue by rhysperry111** [fusionhuh/wava#4]
Hey, just wanna say that I created an AUR package for this project [here](https://aur.archlinux.org/packages/wava-git). You might want to update the PKGBUILD to reflect this.

Have a nice day :)

**Comment by fusionhuh** [comment]
oh, thank you! ill work on this later today

**Comment by fusionhuh** [comment]
hi there, i have a couple questions about the PKGBUILD on the AUR. 
on line 30, it seems to move the config file into /usr/share/wava:

`install -Dm644 "${srcdir}/wava/config/wava.cfg" "${pkgdir}/usr/share/wava/wava.cfg"`

i only designed wava to read config files from ~/.config/wava/ so i didn't have to deal with needing root access to modify the file as well as having the potential for user-specific configurations. i assume that just moving it into ~/.config/wava independent of ${pkgdir} would break standard AUR conventions? if so, i imagine that we would have to elevate the write permissions of /usr/local/wava/ so non-root users running wava could write to the config file, right? i am not familiar with application packaging so perhaps you could give some insight here

**Comment by rhysperry111** [comment]
AUR packages are system wide packages, so creating a directory within the home dir is definitely not allowed.

The reason I copied the config into `/use/share/wava.cfg` is simply so that users have a copy stored somewhere without having to download it from GitHub.

If you look through `wava.install` [here](https://aur.archlinux.org/cgit/aur.git/tree/wava.install?h=wava-git) which is referenced from within the PKGBUILD, I have also set up a message to be displayed to the user when the package is installed telling them that the default config is in `/use/share/wava/` and that they will need to copy it to their home directory.
